app/src/main/java/org/oppia/android/app/topic/lessons/StorySummaryViewModel.kt

```diff
@@ -6,8 +6,30 @@ import org.oppia.android.app.model.StorySummary
 /** [ViewModel] for displaying a story summary. */
 class StorySummaryViewModel(
   val storySummary: StorySummary,
-  private val storySummarySelector: StorySummarySelector
+  private val storySummarySelector: StorySummarySelector,
+  private val chapterSummarySelector: ChapterSummarySelector
 ) : TopicLessonsItemViewModel() {
+  private val arrayList = ArrayList<ChapterSummaryViewModel>()
+
+  val chapterSummaryItemList: List<ChapterSummaryViewModel> by lazy {
+    getRecyclerViewItemList()
+  }
+
+  private fun getRecyclerViewItemList(): ArrayList<ChapterSummaryViewModel> {
```

Comment:
Perhaps instead: ``computeChapterSummaryItemList`` (prefer to be specific vs. 'recycler view item list' and prefer 'compute' since this isn't a trivial getter).

Reply:
done